### PR TITLE
Update ANCM version to 12.1

### PR DESCRIPTION
**Pull request by jkotalik**
For https://github.com/aspnet/IISIntegration/issues/1258

### CODE REVIEW

File: version.props [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <FeatureBranchVersionPrefix Condition="'$(FeatureBranchVersionPrefix)' == ''">a-</FeatureBranchVersionPrefix>
     <VersionSuffix Condition="'$(VersionSuffix)' != '' And '$(FeatureBranchVersionSuffix)' != ''">$(FeatureBranchVersionPrefix)$(VersionSuffix)-$([System.Text.RegularExpressions.Regex]::Replace('$(FeatureBranchVersionSuffix)', '[^\w-]', '-'))</VersionSuffix>
     <VersionSuffix Condition="'$(VersionSuffix)' != '' And '$(BuildNumber)' != ''">$(VersionSuffix)-$(BuildNumber)</VersionSuffix>
-    <AspNetCoreModuleVersionMajor>8</AspNetCoreModuleVersionMajor>
+    <AspNetCoreModuleVersionMajor>12</AspNetCoreModuleVersionMajor>
     <AspNetCoreModuleVersionMinor>1</AspNetCoreModuleVersionMinor>
     <AspNetCoreModuleVersionRevision>0</AspNetCoreModuleVersionRevision>
```
Comment (natemcmaster) [inline version.props]: Will this or the minor version need to change in future servicing updates? If so, I'd suggest tying these to the value of `VersionPrefix`.

**Comment by shirhatti** [comment]
## Description
 
TODO: Describe the bug
 
## Customer Impact
 
MSI engine will fail to upgrade the binary as the current version is lower than a version we've already shipped
 
## Regression from?
 
Regression from when we've moved the source code from SD to GitHub.
 
## Risk
 
Very low. We're just bumping up the assembly version

File: version.props (hunk 1 above)
Comment (jkotalik) [reply version.props]: Nvrm, versions are on a repo by repo basis.

**Comment by jkotalik** [comment]
🆙 📅

File: version.props [hunk 2]
```diff
@@ -1,15 +1,18 @@
 ﻿<Project>
   <PropertyGroup>
-    <VersionPrefix>2.1.2</VersionPrefix>
+    <DotnetMajorVersion>2</DotnetMajorVersion>
```
Comment (natemcmaster) [inline version.props]: Minor nit: we typically capitalize the N in "DotNet".

**Comment by natemcmaster** [comment]
(Wait for shiproom approval, of course)

**Comment by leecow** [comment]
Shiproom approved

**Pull request merged**